sound/soc/sof/sof-client.h

```diff
@@ -39,7 +41,9 @@ int sof_client_ipc_tx_message(struct sof_client_dev *cdev, void *ipc_msg,
 			      void *reply_data, size_t reply_bytes);
 int sof_client_ipc_set_get_data(struct sof_client_dev *cdev, void *ipc_msg,
 				bool set);
-
+#ifdef CONFIG_SND_SOC_SOF_INTEL_IPC4
+struct sof_ipc4_fw_module *sof_client_ipc4_find_module_by_uuid(struct sof_client_dev *cdev, const guid_t *uuid);
+#endif
```

Comment:
this #ifdfef seems unnecessary in a header file, and inconsistent with the inclusion of 'struct sof_ipc4_fw_module' unconditionally above.

Reply:
Indeed, declaration alone does not cause reference. Thanks, I'll fix that.